src/libinjector/linux/linux_utils.c

```diff
@@ -0,0 +1,92 @@
+#include "linux_utils.h"
+#include "linux_debug.h"
+#include <sys/mman.h>
+#include <fcntl.h>
+
+bool setup_exit_syscall(injector_t injector, x86_registers_t* regs, uint32_t no) {
```

Comment:
are we using this?

Reply:
nope, not right now ( and probably not later also "this function" unless we use this for SIGKILL process and create some method like that )

Reply:
Then let's remove it till we need it.